docs/dev_guide/diagnostic_ref/dpct1072.rst

```diff
@@ -23,4 +23,41 @@ device.
 Suggestions to Fix
 ------------------
 
-Try to rewrite the code to not rely on unsupported functionality.
\ No newline at end of file
+Try to rewrite the code to not rely on unsupported functionality.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo() {
+     size_t result1, result2;
+     cudaMemGetInfo(&result1, &result2);
+   }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo() {
+     size_t result1, result2;
+     /*
+     DPCT1072:0: SYCL currently does not support getting the available memory on
+     the current device. You may need to adjust the code.
+     */
+     result2 = dpct::get_current_device().get_device_info().get_global_mem_size();
+   }
+
+which is rewritten to:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo() {
+     size_t result1, result2;

```

Comment:
suggest user to re-migrate with ext feature: device_info

Reply:
done